lib/rgl/bidirectional_adjacency.rb

```diff
@@ -0,0 +1,87 @@
+# bidirectional_adjacency.rb
+#
+require 'rgl/adjacency'
+require 'rgl/bidirectional'
+
+module RGL
+
+  # BGL defines the concept BidirectionalGraph as follows:
```

Comment:
DRY: Do not repeat the comment in the module.

```suggestion
  This implementation of @see BidirectionalGraph ...
```

Reply:
Done.